spec/code_coverage_helper.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+require "simplecov"
+
+unless ENV["DISABLE_SIMPLECOV"] == "true"
+  SimpleCov.start("rails") do
+    add_filter "vendor"
```

Comment:
filter is updated to `vendor` from `spec`. Intentional or?

Reply:
someone didn't read the PR-message 🤔

Reply:
ah.. forgot to read spec, sneaky one 🤔

Reply:
mhh... after a little bit of research I don't think we need to exclude the `spec` folder.
The generated report is identical with or without the filter.

Reply:
Simplecov adds `spec` and `vendor` for us. https://github.com/colszowka/simplecov/blob/master/lib/simplecov/profiles/test_frameworks.rb https://github.com/colszowka/simplecov/blob/4e405d5fe0520a43f6757039444a423fed928c6a/lib/simplecov/profiles/bundler_filter.rb
`vendor` is added by bundler_filter which is added by default. https://github.com/colszowka/simplecov/blob/4e405d5fe0520a43f6757039444a423fed928c6a/lib/simplecov/defaults.rb#L15
`spec` is added by test_framework which is added by the rails profile. https://github.com/colszowka/simplecov/blob/4e405d5fe0520a43f6757039444a423fed928c6a/lib/simplecov/profiles/rails.rb

```suggestion
```

Reply:
uhh nice research 💪

Reply:
My experience is that these are not respected in the version of simplecov used by codeclimate.